server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/validation/MailboxName.java

```diff
@@ -26,11 +26,16 @@
 public class MailboxName {
 
     public static final CharMatcher INVALID_CHARS_MATCHER = CharMatcher.anyOf("%*&#");
+    private static final String CONSECUTIVELY_DOTS = "..";
     private final String mailboxName;
 
     public MailboxName(String mailboxName) {
-        Preconditions.checkArgument(!Strings.isNullOrEmpty(mailboxName));
-        Preconditions.checkArgument(INVALID_CHARS_MATCHER.matchesNoneOf(mailboxName));
+        Preconditions.checkArgument(!Strings.isNullOrEmpty(mailboxName), "MailboxName must not be null or empty");
+        Preconditions.checkArgument(INVALID_CHARS_MATCHER.matchesNoneOf(mailboxName),
+            "MailboxName contain invalid characters");
+        Preconditions.checkArgument(!mailboxName.contains(CONSECUTIVELY_DOTS),
+            "MailboxName contain invalid characters");
```

Comment:
These preconditions are likely an heresy. We should rely on `MailboxPath::assertAcceptable` and handle the resulting `MailboxNameException` rather than duplicating the checks done at the `mailbox-api` level.

This proved to be unreliable.

I am available to help you on this topic.